stl/inc/vector

```diff
@@ -1206,12 +1206,7 @@ public:
 
         _Adl_verify_range(_First, _Last);
         const auto _Whereoff = static_cast<size_type>(_Whereptr - _Oldfirst);
-#ifdef __cpp_lib_concepts
-        constexpr bool _Is_fwd = _Is_fwd_iter_v<_Iter> || forward_iterator<_Iter>;
-#else // ^^^ __cpp_lib_concepts ^^^ / vvv !__cpp_lib_concepts vvv
-        constexpr bool _Is_fwd = _Is_fwd_iter_v<_Iter>;
-#endif // ^^^ !__cpp_lib_concepts ^^^
-        if constexpr (_Is_fwd) {
+        if constexpr (_Is_ranges_fwd_iter_v<_Iter>) {

```

Comment:
Every time someone touches a container, I think "Great, another merge conflict for the P1206R7 PR." And then I realize that _every_ change is a merge conflict for Stephan's Modules PR, and then I feel really bad for Stephan. (No change required.)

Reply:
:joy_cat: The modules PR-to-be is actually much more resistant to merge conflicts than you might expect, partially because git is great at resolving changes that touch different lines, partially because I generally need to modify only the top-level class/function definition introductions.

(Large-scale code movement *is* a headache.)